src/twisted/test/test_persisted.py

```diff
@@ -446,6 +446,12 @@ def test_circularTuple(self):
         self.assertIs(oj[0][0], oj)
         self.assertEqual(oj[0][1], 4321)
 
+    def testIndentify(self):
+        self.assertEqual(
+            aot.jellyToSource({"hello": {"world": []}}),
+            "app={\n  'hello':{\n    'world':[],\n    },\n  }",
+        )
+
```

Comment:
Maybe is easier to read the test... but as long as the test has a docstring, it should be good to merge.

```suggestion
    def testIndentify(self):
        """
        The generated serialization is indented.
        """
        self.assertEqual(
            aot.jellyToSource({"hello": {"world": []}}),
            """app={
  'hello':{
      'world':[],
     },
   }""")

```